addons/dark-www/experimental_scratchr2.css

```diff
@@ -697,6 +697,9 @@ input.link.black {
   background-image: url("%addon-self-dir%/assets/meow.png");
   background-size: 20px 15px;
 }
+.stats li {
+  color: var(--darkWww-box-scratchr2HeaderText);
```

Comment:
There are already selectors for this around line 150. Could this be added there instead?

Reply:
I haven't looked yet, but I'm pretty sure that's actually `.stats li.data`

Reply:
The only thing I found is at line 696:

https://github.com/ScratchAddons/ScratchAddons/blob/215e2c9748bd87823e4b18124818dcecc96671ba/addons/dark-www/experimental_scratchr2.css#L696-L699

And it is `.stats li.data`.

Reply:
I meant `--darkWww-box-scratchr2HeaderText` not the selector itself.

Reply:
I think samq means the `color: var(...)` rule can already be found in the file and we just need to add the `.stats li` selector to it

Reply:
Oh! That makes more sense. Sorry, I'll do that